posthog/models/plugin.py

```diff
@@ -53,3 +53,17 @@ class PluginAttachment(models.Model):
     file_name: models.CharField = models.CharField(max_length=200)
     file_size: models.IntegerField = models.IntegerField()
     contents: models.BinaryField = models.BinaryField()
+
+
+class PluginStorage(models.Model):
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["team_id", "plugin_config_id", "key"], name="posthog_unique_plugin_storage_key"
+            )
+        ]
+
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+    plugin_config: models.ForeignKey = models.ForeignKey("PluginConfig", on_delete=models.CASCADE)
```

Comment:
PluginConfig already has Team as a foreign key, what's the rationale for both `team` and `plugin_config` here?

Reply:
I guess I'm just paranoid about leaking data. Though to be fair, you're right and this can be removed. I'll submit a patch.